src/SFA.DAS.Forecasting.Triggers/Startup.cs

```diff
@@ -58,20 +62,60 @@ public IServiceProvider Build()
         {
             var services = new ServiceCollection();
 
-            services.Configure<ForecastingJobsConfiguration>(Configuration.GetSection("forecastingJobsConfiguration"));
+            services.Configure<ForecastingJobsConfiguration>(Configuration.GetSection("ForecastingJobs"));
 
-            services.AddSingleton(_ =>
-                _loggerFactory.CreateLogger(LogCategories.CreateFunctionUserCategory("Common")));
+            var nLogConfiguration = new NLogConfiguration();
 
-            services.AddSingleton<IEncodingService,EncodingService>();
+            services.AddLogging((options) =>
+            {
+                options.AddConfiguration(Configuration.GetSection("Logging"));
+                options.SetMinimumLevel(LogLevel.Trace);
+                options.AddNLog(new NLogProviderOptions
+                {
+                    CaptureMessageTemplates = true,
+                    CaptureMessageProperties = true
+                });
+
+                options.AddConsole();
+                options.AddDebug();
+
+                nLogConfiguration.ConfigureNLog(Configuration);
```

Comment:
Looks like this line doesn't logically belong in the lambda. Could it be moved to the line after `AddLogging`?